test.sh

```diff
@@ -0,0 +1,14 @@
+echo "#############   Start pack ############"
+# For local package source directory(./src/package/bin/Debug in Nuget.config) should present for restore successful.
+mkdir -p ./src/package/bin/Debug
+dotnet pack -p:"version=1.0.0-dev"
+
+echo "############  Run E2E tests  ############"
+
+dotnet restore -p:"LoggerVersion=1.0.0-dev" --configfile Nuget.config
+dotnet build --no-restore
+dotnet test --no-build ./test/Xunit.Xml.TestLogger.NetCore.Tests/ --logger:"xunit;LogFilePath=test-results.xml"
+dotnet test --no-build ./test/Xunit.Xml.TestLogger.NetFull.Tests/ --logger:"xunit;LogFilePath=test-results.xml"
+
+# Todo add validation 
```

Comment:
Did we consider doing these steps in a test method? We could invoke *dotnet build* with `dotnet build --source c:\packages\mypackages` in order to pick up the built nuget package from `./src/package/bin/Debug`.

So our CI step just becomes `dotnet pack; dotnet test`. It will run the test method for all validation. We need not author any os specific scripts.

Reply:
This is still pending.